iidm/iidm-tck/src/test/java/com/powsybl/iidm/network/tck/extensions/AbstractStandbyAutomatonTest.java

```diff
@@ -68,6 +66,12 @@ public void test() {
             fail();
         } catch (Exception ignored) {
         }
+
+        // when standby is false do not throw error on inconsistent low and high voltage thresholds use case
+        standbyAutomaton.setStandby(false);
+        assertFalse(standbyAutomaton.isStandby());
+        standbyAutomaton.setHighVoltageThreshold(200f);
+        assertEquals(200f, standbyAutomaton.getHighVoltageThreshold(), 0.0);
```

Comment:
We could add an assertion checking that setting the standBy mode when data are inconsistent leads to an exception:
```suggestion
        assertEquals(200f, standbyAutomaton.getHighVoltageThreshold(), 0.0);
        
        // but an exception is thrown when the standby mode is latter used
        assertThrows(IllegalArgumentException.class, () -> standbyAutomaton.setStandby(true));
```